src/main/java/ratpack/zipkin/ServerTracingModule.java

```diff
@@ -33,8 +33,8 @@
 import ratpack.zipkin.internal.DefaultServerTracingHandler;
 import ratpack.zipkin.internal.RatpackCurrentTraceContext;
 import ratpack.zipkin.internal.ZipkinHttpClientImpl;
-import zipkin.Endpoint;
-import zipkin.Span;
+import zipkin2.Endpoint;
+import zipkin2.Span;
 import zipkin.reporter.Reporter;
```

Comment:
zipkin2.reporter.Reporter

Reply:
`Config#spanReporter` needs to be updated to return `zipkin2.reporter.Reporter` too. cc/ @llinder

Reply:
I suppose this would mean that v1 `Reporter`s aren't going to be usable anymore, which would make this a breaking change.